lib/util/utils.ts

```diff
@@ -386,4 +386,13 @@ export function applyMixins(targetCtor: any, sourceCtors: any[]): void {
       targetCtor.prototype[name] = sourceCtors.prototype[name];
     });
   });
-}
\ No newline at end of file
+}
+
+const validateISODuration = /^(-|\+)?P(?:([-+]?[0-9,.]*)Y)?(?:([-+]?[0-9,.]*)M)?(?:([-+]?[0-9,.]*)W)?(?:([-+]?[0-9,.]*)D)?(?:T(?:([-+]?[0-9,.]*)H)?(?:([-+]?[0-9,.]*)M)?(?:([-+]?[0-9,.]*)S)?)?$/;
+
+/**
+ * Indicates whether the given string is in ISO 8601 format.
```

Comment:
/*
 * Indicates whether the given string is in ISO 8601 format.
 * @param {string} value - The value to be validated for ISO 8601 duration format.
 * @return {boolean} result - `true` if valid, `false` otherwise.
 */
jsdoc comment format, since this will be a public facing method and will end up on docs.microsoft.com